### PR TITLE
Added new courses in free-courses-bn.md

### PR DESCRIPTION
**Pull request by Akash190104**
## What does this PR do?
Add resource(s) 
Added courses for Next.js, Flask, Linux, Swift, and AngularJS
### Why is this valuable (or not)?
These are handpicked youtube tutorial courses for beginners to learn more about programming.
### How do we know it's really free?
They are public youtube playlists
### For book lists, is it a book? For course lists, is it a course? etc.
Course
## Checklist:
- [ X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ]X [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ X] Include author(s) and platform where appropriate.
- [ X] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -16,7 +16,9 @@
 * [IDE and editors](#ide-and-editors)
 * [Java](#java)
 * [JavaScript](#javascript)
+    * [AngularJS](#angularJS)
```
Comment (eshellman) [inline courses/free-courses-bn.md:19]:
```suggestion
    * [AngularJS](#angularjs)
```

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -16,7 +16,9 @@
...
     * [jQuery](#jquery)
+    * [Next.js](#next.js)
```
Comment (eshellman) [inline courses/free-courses-bn.md:21]:
```suggestion
    * [Next.js](#nextjs)
```

**Review by eshellman — CHANGES_REQUESTED** [review]

**Comment by Akash190104** [comment]
@eshellman Please review.

**Comment by Akash190104** [comment]
@eshellman Could you please review?

File: courses/free-courses-bn.md [hunk 3]
```diff
@@ -16,7 +16,9 @@
 * [IDE and editors](#ide-and-editors)
 * [Java](#java)
 * [JavaScript](#javascript)
+    * [AngularJS](#angularjs)
```
Comment (eshellman) [inline courses/free-courses-bn.md:19]: The heading is Angular, not AngularJS, so we got this wrong. I prefer `#angular`, although I'm not positive this is correct. either way, the heading and the index need to match.

Comment (Akash190104) [reply courses/free-courses-bn.md]: @eshellman Did the necessary changes. Is this fine now?

**Pull request merged**